packages/website/plugins/add-rule-attributes-list.ts

```diff
@@ -13,191 +13,25 @@ const addRuleAttributesList: Plugin = () => {
     if (rule == null) {
       return;
     }
-    const config = ((): 'recommended' | 'strict' | null => {
-      switch (rule.meta.docs?.recommended) {
-        case 'error':
-        case 'warn':
-          return 'recommended';
-
-        case 'strict':
-          return 'strict';
-
-        default:
-          return null;
-      }
-    })();
-    const autoFixable = rule.meta.fixable != null;
-    const suggestionFixable = rule.meta.hasSuggestions === true;
-    const requiresTypeInfo = rule.meta.docs?.requiresTypeChecking === true;
```

Comment:
Since we are making the rule meta available as a global data context, I decided to migrate the rule attributes to use the same infrastructure as well to make the best use of it. This way there's less client code generated overall.